tests/setup-test.js

```diff
@@ -1,19 +1,73 @@
 'use strict';
 
 var tape = require('tape');
-var server = require('../example/server.js');
+var init = require('../example/server.js');
+var tagsPool = require('../example/pg.js')();
+var tags = require('../example/tags.json');
+var server;
+
+// Tests assume we have a table called tags which is empty
+
+
+// First check that the DB is empty
+tape('check that the DB is empty', function (t) {
```

Comment:
I'm not sure about this test. If we want to make sure that the database is clean, could we just run a script which initialise a test database before running the tests? Something like [this](https://github.com/dwyl/hapi-login-example-postgres/blob/master/test/_create_table.js)